src/enr/crypto.ts

```diff
@@ -0,0 +1,63 @@
+// @ts-ignore
+import keccak from "keccakjs";
+import randomBytes from "randombytes";
+import * as secp256k1 from "secp256k1";
```

Comment:
both keccak and secp256k1 are already in bcrypto.

Maybe we don't need all those crypto libraries, not sure where else bcrypto is used

Reply:
also apparently random bytes :joy: now we just need types -- will investigate!